.github/workflows/test.yml

```diff
@@ -100,7 +100,18 @@ jobs:
           go env -u CC
           go env -u CXX
 
-      # TODO: add Windows 386 tests
+      - name: go test (Windows 386)
+        if: runner.os == 'Windows'
+        run: |
+          env CGO_ENABLED=0 GOARCH=386 go test -shuffle=on -v -count=10 ./...
+          env CGO_ENABLED=1 GOARCH=386 go test -shuffle=on -v -count=10 ./...
+
+      - name: go test (Linux 386)
+        if: runner.os == 'Linux'
+        run: |
+          go env -w CC=i386-linux-gnu-gcc
+          go env -w CXX=i386-linux-gnu-g++
+          env CGO_ENABLED=1 GOOS=linux GOARCH=386 go test -shuffle=on -v -count=10 ./...
```

Comment:
Would this work?

https://github.com/hajimehoshi/ebiten/blob/af4638d83f6a4714d8e637ec4545fc58ce5ceb7a/.github/workflows/test.yml#L142-L151